modules/creating-an-infra-node.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/cluster-tasks.adoc
+
+[id="creating-an-infra-node_{context}"]
+= Creating an infra node
+
+[IMPORTANT]
+====
+This procedure applies to user-provisioned infrastructure environments only.
+
+See Creating infrastructure machine sets for installer-provisioned infrastructure
+environments or for any cluster where the master nodes are managed by the
+machine API.
+====
+
+Requirements of the cluster dictate that infra node(s) be provisioned. The
+installer only provides provisions for master and worker nodes. Worker nodes can
+be designated as infra nodes or app nodes through labeling.
+
+.Procedure
+
+. Add a label to the worker node(s) that you want to act as app node(s):
++
+[source,terminal]
+----
+$ oc label node <node-name> node-role.kubernetes.io/app=""
+----
+
+. Add a label to the worker node(s) that you want to act as infra node(s):
++
+[source,terminal]
+----
+$ oc label node <node-name> node-role.kubernetes.io/infra=""
+----
+
+. Check to see if applicable nodes now have the infra role and app roles:
++
+[source,terminal]
+----
+$ oc get nodes
```

Comment:
Not critical, but is it possible to give example output of what they should expect to see?